transfer-repo-into-the-org.md

```diff
@@ -4,16 +4,18 @@
 
 Ideally, the repository should have the following documents in place:
 
-- `CODE_OF_CONDUCT.md`: it can be a reference to
-  [the Node.js Code of Conduct][coc].
-- `CONTRIBUTING.md`: if there isn't one already, [the contributing guide][]
-  of Node.js core could be a good example. Consider including the Developer's
-  Certificate of Origin section in the document to avoid potential copyright
-  conflicts.
 - `LICENSE`, or other kind of documents that describe the license of
   the project.
 - `README.md`
 
+Ideally, it would not have the following documents to inherit them from
```

Comment:
I think there was some discussion that when you clone a repo that inherits like this you don't get a copy of the files and that might be a concern.

I don't remember if/where that concern was discussed/addressed but it's probably good to document that it's understood that there will not be copies in case of a clone (unless that has changed or I've got that wrong) and its a choice that we have made on purpose.

@nodejs/tsc any concerns with the code of conduct or contributing files not showing up in a clone?

Reply:
You are correct that you do not get a copy of the Code of Conduct checked into the repo.

Reply:
At this point, the `.github` pattern is pretty prevalent and I don't think people *expect* the files to be always present.